test/artifact_test.js

```diff
@@ -5,14 +5,13 @@ suite('Artifacts', function() {
   var assert        = require('assert');
   var slugid        = require('slugid');
   var _             = require('lodash');
-  var Promise       = require('promise');
-  var request       = require('superagent-promise');
+  var request       = require('superagent');
   var assert        = require('assert');
   var urljoin       = require('url-join');
   var BlobUploader  = require('./azure-blob-uploader-sas');
-  var Bucket        = require('../lib/bucket');
-  var BlobStore     = require('../lib/blobstore');
-  var data          = require('../lib/data');
+  var Bucket        = require('../src/bucket');
```

Comment:
isn't this just `./` instead of `../`

Reply:
The paths are relative to the source file, `tests/artifact_test.js`.  So `./src` wouldn't be found.